packages/relay-kit/src/packs/safe-4337/types.ts

```diff
@@ -50,6 +54,8 @@ export type Safe4337InitOptions = {
   }
   options: ExistingSafeOptions | PredictedSafeOptions
   paymasterOptions?: PaymasterOptions
+  // on-chain analitics
```

Comment:
The variable name does the same as the comment right? and the other props has no comments

Reply:
true, removed! 🙏